_posts/2023-12-05-release.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Release 2023.49
+author: Priya
+date: 2023-12-05
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+###Export as Dataset (Alpha Feature):
+Mammoth now allows you to export data to a new dataset, overcoming challenges of copying data within Mammoth and avoiding issues with circular dependencies or linked deletions.
```

Comment:
You can now create a Dataset from a View as a one-time export. This feature supports exporting data based on filters in explore cards. Since this is a one-time export, it is disjointed and not added to pipelines in the View.